### DISCUSSION TITLE
Support partitioned tables

### DISCUSSION THREAD

**Discussion by belovaf** [diesel-rs/diesel#4091] [Ideas]
Partitioned tables can be without PK. PK will be defined on each partition table.
Diesel should provide a filter to exclude partition tables from a schema.

**Comment by weiznich** [comment]
Diesel-CLI already allows you to specify which tables should be included/excluded while executing `print-schema`. See the [documentation for more details](http://diesel.rs/guides/configuring-diesel-cli.html). Given that it's not clear what you propose to add here.

**Reply by belovaf** [reply]
This doesn' work with partitioned tables.
If I add the main table to include list, partitioned tables are also included regardless of filters.

**Reply by weiznich** [reply]
In that case I'm happy to accept a PR for this as long as it comes with tests. Given that I personally do not use partitioned tables it's not on my priority list to fix that.
